### DISCUSSION TITLE
Allow payments for orders that must be approved

### DISCUSSION THREAD

**Discussion by Akhun-Delar** [pretix/pretix#3290] [Ideas]
I am planning to work with a blacklist of people that shouldn't attend my events, so I would be using the function to approve orders before the process starts. In my opinion, it should be possible to accept payments before that approval, so there is no ticket / no confirmation, before I manually (bulk) approve them.

**Comment by raphaelm** [comment]
Can't you just cancel orders from banned people?

**Reply by Akhun-Delar** [reply]
My goal is that those people don't get a ticket generated in the first place. :/ I guess the people that are on our blacklist would just show up with their ticket and tell us that they didn't get the cancellation email
